src/_guides/environment-declarations.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Configuring apps with environment declarations
+description: >
+  Learn about using environment declarations to customize application behavior.
+---
+
+Dart supports specifying environment declarations when
+building or running an application.
+Environment declarations are used to specify
+configuration values or options as key-value pairs
+that are accessed and evaluated at compile time.
+
+Your app can use the values of environment declarations
+to change its functionality or behavior.
+Dart compilers can take advantage of this to eliminate code
+no longer reached due to the values specified through tree shaking.
```

Comment:
Comma before "through", otherwise it sounds like the values are specified through tree shaking.

Consider quoting "tree shaking", readers are not necessarily expected to know what it means.